Modules/SubModule_Import/xASL_wrp_NII2BIDS_Subject.m

```diff
@@ -1,13 +1,13 @@
-function x = xASL_wrp_NII2BIDS_Subject(x, bidsPar, studyPar, nameSubjectSession)
+function x = xASL_wrp_NII2BIDS_Subject(x, bidsPar, studyParFull, nameSubjectSession)
 %xASL_wrp_NII2BIDS_Subject Run NII to ASL-BIDS for one individual subject.
 %
-% FORMAT: x = xASL_wrp_NII2BIDS_Subject(x, bidsPar, studyPar, nameSubjectSession)
+% FORMAT: x = xASL_wrp_NII2BIDS_Subject(x, bidsPar, studyParFull, nameSubjectSession)
 % 
 % INPUT:
 %   x                      - ExploreASL x structure (REQUIRED, STRUCT)
 %   x.modules.import.imPar - JSON file with structure with import parameter (REQUIRED, STRUCT)
 %   bidsPar                - Output of xASL_imp_Config (REQUIRED, STRUCT)
-%   studyPar               - JSON file with the BIDS parameters relevant for the whole study (REQUIRED, STRUCT)
+%   studyParFull           - JSON file with the BIDS parameters relevant for the whole study, potentially multi-context (REQUIRED, STRUCT)
```

Comment:
"potentiall multi-context" -> "supports providing nested studyPars for all potential subject/session/run permutations/combinations"

Reply:
fixed